test/test-integration/src/test/java/com/alipay/sofa/rpc/server/rest/CustomizeFilterTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.server.rest;
+
+import com.alipay.sofa.rpc.common.RpcConstants;
+import com.alipay.sofa.rpc.config.*;
+import com.alipay.sofa.rpc.filter.Filter;
+import com.alipay.sofa.rpc.test.ActivelyDestroyTest;
+import junit.framework.Assert;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author zhangchengxi
+ * Date 2018/12/2
+ */
+public class CustomizeFilterTest extends ActivelyDestroyTest {
+
+    static RestService                 filterRestService;
+
+    static CustomizeTestFilter         providerFilter;
+
+    static CustomizeTestFilter         clientFilter;
+    static ProviderConfig<RestService> providerConfig;
+
+    static {
+        JAXRSProviderManager.registerCustomProviderInstance(new CustomizeContainerRequestTestFilter());
+        JAXRSProviderManager.registerCustomProviderInstance(new CustomizeContainerResponseTestFilter());
+        JAXRSProviderManager.registerCustomProviderInstance(new CustomizeClientRequestTestFilter());
+        JAXRSProviderManager.registerCustomProviderInstance(new CustomizeClientResponseTestFilter());
```

Comment:
use `@beforeClass` method for `registerCustomProviderInstance()`, and unregister them by `@afterClass` method.